build.sh

```diff
@@ -155,5 +155,12 @@ configure_user(){
     echo '%wheel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers.d/wheel-nopasswd
 }
 
+write_repo() {
+    # shellcheck source=src/cmdlib.sh
+    . "${srcdir}/src/cmdlib.sh"
+    mkdir -p /lib/coreos-assembler/src
+    prepare_git_artifacts /root/containerbuild /lib/coreos-assembler/src/archive.tar.gz /lib/coreos-assembler/src/git.json
```

Comment:
can we name these something more specific. Originally I didn't understand that we are calling `prepare_git_artifacts()` once at container build time to capture the git information about the assembler and once at assembler runtime (during `coreos-assembler build`) to capture information about the config repository.

Something like:
- `coreos-assembler-archive.tar.gz`
- `coreos-assembler-git.json`

maybe

Reply:
Implemented.